cosmo_tester/framework/test_hosts.py

```diff
@@ -864,16 +865,20 @@ def _save_manager_logs(self):
                                   '{}, since server has been deleted or not '
                                   'initialized.'.format(i))
             else:
-                self._save_logs_for_instance(instance, logs_dir)
+                self._save_logs_for_instance(instance, logs_dir, i)
 
         self._logger.debug('_save_manager_logs completed')
 
     @retry(stop_max_attempt_number=3, wait_fixed=5000)
-    def _save_logs_for_instance(self, instance, logs_dir):
+    def _save_logs_for_instance(self, instance, logs_dir, instance_index):
         self._logger.info('Attempting to download logs for Cloudify Manager '
                           'with ID: {}...'.format(instance.server_id))
         instance.use()
-        logs_filename = '{}_logs.tar.gz'.format(instance.server_id)
+        prefix = '{}__'.format(instance.__class__.__name__)
+        if self._request:
+            prefix += 'pytestparam_{}__'.format(self._request.param)
+        prefix += 'index_{}__'.format(instance_index)
+        logs_filename = '{}__{}_logs.tar.gz'.format(prefix, instance.server_id)
```

Comment:
What is the desired result?

Reply:
Knowing which logs belongs to which fixture inside the test